amundsen_application/static/js/components/common/Table/index.tsx

```diff
@@ -68,6 +74,15 @@ type EmptyRowProps = {
 const getCellAlignmentClass = (alignment: TextAlignmentValues) =>
   ALIGNEMENT_TO_CLASS_MAP[alignment];
 
+const fieldIsDefined = (field, row) => row[field] !== undefined;
+const checkValidData = (data, fields) => {
+  fields.forEach((field) => {
+    if (data.filter(fieldIsDefined.bind(null, field)).length === 0) {
+      throw new Error(INVALID_DATA_ERROR_MESSAGE);
+    }
+  });
+};
```

Comment:
I would suggest something along the lines of:
```suggestion
type Some = string | number | boolean | symbol | bigint | object; 
interface ValidData =   Record<string, Some> // to sass out the undefined | null;
// guard
const checkValidData = (data: unknown[], fields): is ValidData[] => {
   return data.filter(fieldIsDefined.bind(null, field)).length === 0
}
```

Reply:
to read more about guards, you can go [here](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards)

Reply:
Thanks man!